INSTALL.md

```diff
@@ -101,6 +109,13 @@ make
 make install
 ```
 
+You will also need to install pyroute2 to be able to run `make test` on BCC:
+```
+git clone https://github.com/svinota/pyroute2.git
+cd pyroute2
```

Comment:
Lighten the tone to something like "If you're planning on running BCC self-tests with the `make test` command, you may also need to install `pyroute2`". I guess not everyone wants to do a `make test` so good to not make it sound mandatory.

Reply:
Good point. Done.